packages/bpk-docs/src/components/neo/InfoPageBuilder/InfoPageBuilder.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+/* @flow */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import marked from 'marked';
+import BpkAnimateHeight from 'bpk-animate-height';
+import BpkButton from 'bpk-component-button';
+import BpkContentContainer from 'bpk-component-content-container';
+import { cssModules } from 'bpk-react-utils';
+import ChevronDownIcon from 'bpk-component-icon/lg/chevron-down';
+import { withButtonAlignment } from 'bpk-component-icon';
+
+import HeroSection from '../HeroSection';
+import Heading from '../Heading';
+import STYLES from './InfoPageBuilder.scss';
+import AlternatingPageContent from '../AlternatingPageContent';
+
+const getClassName = cssModules(STYLES);
+const renderer = new marked.Renderer();
+const AlignedChevronDownIcon = withButtonAlignment(ChevronDownIcon);
+
+const markdownToHTML = (input: string): string =>
+  marked(
+    input
+      .replace(/^#### /gm, '##### ') // replace h4 with h5
+      .replace(/^### /gm, '#### ') // replace h3 with h4
+      .replace(/^## /gm, '### ') // replace h2 with h3
+      .replace(/^# /gm, '## '), // replace h1 with h2
+    { renderer },
+  );
+
+type SectionType = {
+  id: string,
+  title: string,
+  markdown: string,
+};
+
+type SectionProps = {
+  ...SectionType,
+  expanded: boolean,
+};
+
+type InfoPageBuilderProps = {
+  hero: any,
+  sections: Array<SectionType>,
+  title: string,
+};
+
+class Section extends React.Component<SectionProps, { expanded: boolean }> {
+  static propTypes = {
+    expanded: PropTypes.bool,
+    id: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    markdown: PropTypes.string.isRequired,
+  };
+
+  static defaultProps = {
+    expanded: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = { expanded: this.props.expanded };
+  }
+
+  toggleExpand = () => {
+    this.setState({ expanded: !this.state.expanded });
+  };
+
+  render() {
+    const { id, markdown, title } = this.props;
+    const { expanded } = this.state;
+    return (
+      <section>
+        <button
```

Comment:
Functions as a button - I took this from the accordion component so it should be ok for a11y.